_targets.R

```diff
@@ -0,0 +1,38 @@
+library(targets)
+source("data_extraction_ms.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker","readr","dplyr","stringr"))
```

Comment:
@jesse-ross I added more packages that the example `_targets.R`

Reply:
Perfect! Also, now that those libraries are sourced here, you no longer need to source them in `1_fetch/src/data_extraction_ms.R` or `2_process/src/ModelPlotting_ms.R`.